src/infra/drive-server-wip/services/folders/rename-folder.ts

```diff
@@ -28,26 +28,17 @@ export async function renameFolder(context: { uuid: string; name: string; worksp
   const res = await clientWrapper({
     promiseFn,
     key,
-    loggerBody: {
-      msg: 'Rename folder request',
-      context,
-      attributes: {
-        method,
-        endpoint,
-      },
-    },
+    loggerBody: { msg: 'Rename folder request', context },
   });
 
-  if (res.error?.code === 'UNKNOWN') {
+  if (res.error) {
     switch (true) {
       case res.error.response?.status === 409:
         return { error: new RenameFolderError('FOLDER_ALREADY_EXISTS', res.error.cause) };
+      default:
+        return { error: res.error };
     }
```

Comment:
It would be clearer to replace the switch (true) with an if/else. In this case, you are only evaluating the status of the response, and the if makes it more readable.

```suggestion
    if (res.error.response?.status === 409) {
      return { error: new RenameFolderError('FOLDER_ALREADY_EXISTS', res.error.cause) };
    }
    return { error: res.error };
```